start-fresh.bat

```diff
@@ -0,0 +1,4 @@
+call conda deactivate &&^
+yarn conda:delete &&^
+yarn git:clean &&^
+call ./bootstrap-with-conda-env.bat
```

Comment:
Just ran this here on the work PC, it worked, and so too did `nbstripout` hmm...